windows/src/developer/kmcmpdll/Compiler.cpp

```diff
@@ -1493,10 +1501,11 @@ DWORD CheckStatementOffsets(PFILE_KEYBOARD fk, PFILE_GROUP gp, PWSTR context, PW
 DWORD CheckUseStatementsInOutput(PWSTR output) {   // I4867
   BOOL HasUse = FALSE;
   PWSTR p;
-  for(p = output; *p; p = incxstr(p)) {
-    if(*p == UC_SENTINEL && *(p+1) == CODE_USE) {
+  for (p = output; *p; p = incxstr(p)) {
+    if (*p == UC_SENTINEL && *(p + 1) == CODE_USE) {
       HasUse = TRUE;
-    } else if(HasUse) {
+    }
+    else if (HasUse) {
```

Comment:
Is there some reason for breaking our standard if-else convention at numerous places within this PR?  Seems odd to be changing this now.

```suggestion
    } else if (HasUse) {
```

Reply:
Visual Studio auto format did this and I forgot to undo... :)